src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/WebSocketBuffer.cs

```diff
@@ -167,7 +165,7 @@ internal Interop.WebSocket.Property[] CreateProperties(bool useZeroMaskingKey)
                 properties[3] = new Interop.WebSocket.Property()
                 {
                     Type = WebSocketProtocolComponent.PropertyType.DisableMasking,
-                    PropertySize = (uint)s_SizeOfBool,
+                    PropertySize = (uint)sizeof(uint),
```

Comment:
It needs to be `sizeof(uint)` because of bool gets marshalled as uint. Notice that the "bool" value is written as int32 a few lines above.